microshift_release_notes/microshift-4-18-release-notes.adoc

```diff
@@ -162,3 +162,23 @@ See the latest images included with {microshift-short} by xref:../microshift_upd
 ==== Bug fix
 
 * Previously, a failed greenboot health check flag did not clear when an RPM-install-based {microshift-short} service was updated without rebooting the system. As a consequence, greenboot health checks for optional components continue to fail because the `systemctl restart greenboot-healthcheck.service` command fails while the flag is present. With this release, the primary {microshift-short} greenboot health check clears the condition that causes optional component health checks to continue to fail. (link:https://issues.redhat.com/browse/OCPBUGS-51198[*OCPBUGS-51198*])
+
+[id="microshift-4-18-11-dp_{context}"]
+=== RHBA-2025:4249 - {microshift-short} 4.18.11 bug fix and enhancement advisory
+
+Issued: 1 May 2025
+
+{product-title} release 4.18.11 is now available, see the link:https://access.redhat.com/errata/RHBA-2025:4249[RHBA-2025:4249] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:4211[RHSA-2025:4211] advisory.
+
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-18-11-bugs_{context}"]
+==== Bug fixes
+
+//* Previously, sos reports were missing firewalld information. With this release, the {microshift-short} sos report includes firewalld information for enhanced troubleshooting. (link:https://issues.redhat.com/browse/OCPBUGS-55002[*OCPBUGS-55002*])
+
+* Previously, {microshift-short} could fail to start on a host without a hostname installed. With this release, the hostname package is pulled as a dependency during installation. This change means that the hostname is now required when building the {microshift-short} system, preventing this type of startup failure. (link:https://issues.redhat.com/browse/OCPBUGS-54239[*OCPBUGS-54239*])
+
+* In some cases, {microshift-short} did not start when using the default configuration file shipped in the {microshift-short} RPM. Certain required values were missing from the configuration file. These values are now added to the default `config.yaml` to avoid this problem and {microshift-short} starts as expected. (link:https://issues.redhat.com/browse/OCPBUGS-53026[*OCPBUGS-53026*])
```

Comment:
```suggestion
* In some cases, {microshift-short} did not start when using the default configuration file shipped in the {microshift-short} RPM due to certain required values that were missing. These values are now added to the default `config.yaml` file to avoid this problem and {microshift-short} starts as expected. (link:https://issues.redhat.com/browse/OCPBUGS-53026[*OCPBUGS-53026*])
```